library/testing/TestSDKInteractorCommand.cxx

```diff
@@ -24,6 +24,12 @@ int TestSDKInteractorCommand(int argc, char* argv[])
   inter.triggerCommand("reset render.hdri.file");
   test("triggerCommand reset optional", options.render.hdri.file.has_value() == false);
 
+  // Test clear
+#if F3D_MODULE_UI
+  inter.triggerCommand("print_scene_info");
+  test("triggerCommand clear", (inter.triggerCommand("clear") == true));
+#endif
```

Comment:
While this is an ok way to test this, a baseline based test (in application/testing/CMakeLists.txt) would be better imo.